aten/src/ATen/Context.h

```diff
@@ -135,12 +135,12 @@ static inline void init() {
   }
 }
 
-static inline Type& getType(Backend p, ScalarType s) {
-  return globalContext().getType(p, s);
+static inline Type& getType(Backend p, ScalarType s, bool is_variable = false) {
```

Comment:
prefer no default values (for the same reason as https://github.com/pytorch/pytorch/pull/11017).